src/plugins/tracking/util/clickState.js

```diff
@@ -0,0 +1,56 @@
+import JSCookie from 'js-cookie';
+
+/*
+ * This module provides utilities for managing click tracking data in Redux
+ * state an saving that data to a browser cookie
+ */
+
+export const COOKIE_NAME = 'click-track'; // must remain in sync with Meetup Classic implementation
+
+const BrowserCookies = JSCookie.withConverter({
+	read: (value, name) => value,
```

Comment:
ok - I was sort of thinking that it might be useful for future maintainers to know that these callbacks do have access to the cookie name if needed, but that's easy to find out from the JSCookie docs if this needs to be tweaked for some reason.